docs/Installing-Bouncy-Castle.md

```diff
@@ -1,169 +1,3 @@
 # Installing Bouncy Castle
 
 Since PowerAuth Server in version 1.1.x, installing the Bouncy Castle in OS or Java container is no longer required. The latest version is bundled with the app and cryptographic primitives should work out of the box.
-
```

Comment:
I believe that we may remove it. It is available in the documentation for older versions.